ces_client/utils.py

```diff
@@ -65,4 +68,20 @@ def make_answers(self, state):
         answers.append({'text': 'Goodbye!',
                         'value': settings.DECISIONTREE_SESSION_END_TRIGGER})
 
-        return answers
\ No newline at end of file
+        return answers
+
+    @classmethod
+    def prep_text_for_botui(self, text):
+        '''
+        This function prepares the message text (as returned from the database)
+        for the BotUI object (see: index.html and bot-utils.js).
+
+        BotUI can handle html. So, this function replaces `\n` with 
+        the `<br>` tag.
+
+        The web interface also requires users to "click" rather than "type."
+        This function changes the text accordingly.
+        '''
+        text_with_br = text.replace('\n', '<br>')
+
+        return text_with_br.replace('Type', 'Click')
```

Comment:
i think this would be a little easier to follow if this replacement happened on its own line and the return returned a variable